cmd.go

```diff
@@ -191,8 +192,15 @@ func (ctx *Context) Setenv(key, value string) error {
 }
 
 // AbsPath returns an absolute representation of path, with relative paths
-// interpreted as relative to ctx.Dir.
+// interpreted as relative to ctx.Dir and with "~/" replaced with users
+// home dir.
 func (ctx *Context) AbsPath(path string) string {
+	if len(path) >= 2 && path[:2] == "~/" {
```

Comment:
juju/utils.NormalizePath handles both ~/foo and ~user/foo. We should probably use that?
https://github.com/juju/utils/blob/master/file.go#L42